db/database.go

```diff
@@ -408,16 +409,17 @@ func NewDatabaseContext(ctx context.Context, dbName string, bucket base.Bucket,
 		return nil, err
 	}
 	dbContext := &DatabaseContext{
-		Name:           dbName,
-		UUID:           cbgt.NewUUID(),
-		MetadataStore:  metadataStore,
-		Bucket:         bucket,
-		StartTime:      time.Now(),
-		autoImport:     autoImport,
-		Options:        options,
-		DbStats:        dbStats,
-		CollectionByID: make(map[uint32]*DatabaseCollection),
-		ServerUUID:     serverUUID,
+		Name:                dbName,
+		UUID:                cbgt.NewUUID(),
+		MetadataStore:       metadataStore,
+		Bucket:              bucket,
+		StartTime:           time.Now(),
+		autoImport:          autoImport,
+		Options:             options,
+		DbStats:             dbStats,
+		CollectionByID:      make(map[uint32]*DatabaseCollection),
+		ServerUUID:          serverUUID,
+		UserFunctionTimeout: 60 * time.Second,
```

Comment:
It's preferable to leave this defined as a constant and reference it here (i.e. change existing constant to DefaultUserFunctionTimeout).